### ISSUE TITLE
[Security fix] Add pyxis-api-token-path for those who don't want to disclose the token during the Preflight run

### ISSUE THREAD

**Issue by tkrishtop** [redhat-openshift-ecosystem/openshift-preflight#692]
## Is your feature request related to a problem? Please describe.
During the `preflight check container` run with the `--submit` option, the `pyxis-api-token` is displayed in the console. That exposes the token in the history search, and in the logs of different CI tools.

## Describe the solution you'd like.
It would be nice to deal with the Pyxis token the same way as we deal with the docker config: instead of providing the token itself, provide a path to the file that contains the token.  

To keep backward compatibility, that could be implemented as an additional option `--pyxis-api-token-path`.

## Additional context.
If you're still accepting the contributors, I would be happy to work on the implementation.

**Comment by acornett21** [comment]
I think it might make more sense to use the `config` file approach that is built into viper (and I think still works after our last config refactor, would need to check). Basically all of the args/configs go into a file instead, I don't remember what the syntax of that file looks like, but we can figure out and document.

**Comment by tkrishtop** [comment]
> I think it might make more sense to use the config file approach that is built into viper

Yes, I like the idea, that could be a good proper solution, better than the one I proposed.

> and I think still works after our last config refactor, would need to check

The last time I checked it, it was pretty confusing. Something like - if we want to define the `PFLT_DOCKERCONFIG` env variable (capital letters + PFLT_ prefix) when it should be defined as `dockerconfig` in the config.yaml (lowercase letters + no prefix). I didn't manage to make it work in the past, have to try again.
